tests/testthat/test-utilities-simulation.R

```diff
@@ -155,6 +155,22 @@ test_that("It runs one individual simulation with simulationRunOptions", {
   expect_true(isOfType(results, "SimulationResults"))
 })
 
+test_that("It throws an erro when trying to run multiple simulations", {
```

Comment:
error not erro

Reply:
should we also maybe have a more explicit message. e.g. runSimulation is  a convenience method for better user experience and can only be called with a single simulation. To simulate multiple simulations in parallel, use runSimulations

Reply:
@PavelBal Going to fix the typo and merge